test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageIdValidatorTest.cs

```diff
@@ -18,6 +18,23 @@ public void ValidatePackageIdInvalidIdThrows()
             ExceptionAssert.ThrowsArgumentException(() => PackageIdValidator.ValidatePackageId(packageId), "The package ID '  Invalid  . Woo   .' contains invalid characters. Examples of valid package IDs include 'MyPackage' and 'MyPackage.Sample'.");
         }
 
+        [Fact]
+        public void ValidatePackageId_OnGB18030Characters_Succeeds()
```

Comment:
Test name ends with "Succeeds" but the Act & Assert step does not validate that anything succeeds. It actually only asserts if an exception is thrown, and in that case, verifies that a failure occurred. Can we update the suffix to describe what we expect to assert in this test?

Reply:
`ValidatePackageId` method throws an exception if the id is valid otherwise it does not. In this test case, I am trying to assert that an exception is not thrown when using GB18030 characters. The idea is to use a try catch. Try running `ValidatePackageId` and if it throws an exception we have an Assert.Fail in the catch block.